src/PowerShellEditorServices/Utility/ThreadSynchronizationContext.cs

```diff
@@ -25,6 +25,11 @@ public class ThreadSynchronizationContext : SynchronizationContext
 
         #region Constructors
 
+        public ThreadSynchronizationContext()
+        {
+            var hash = this.GetHashCode();
```

Comment:
Ahhh, I forgot to take that out.  I was debugging a gnarly threading issue when I first added the "attach" debug mode; the old ThreadSynchronizationContext wasn't being cleared out properly so it broke future debugging sessions.